cfme/tests/configure/test_access_control.py

```diff
@@ -2888,3 +2889,40 @@ def test_default_tenat_name_rail_console():
             2. tenant name should be updated on rail console
     """
     pass
+
+
+@pytest.mark.meta(blockers=[BZ(1803952, forced_streams=["5.10"])])
+@test_requirements.rbac
+@pytest.mark.tier(2)
+@pytest.mark.meta(automates=[1803952])
+def test_select_edit_group(appliance):
+    """
+    Should be able to see "edit group" option for existing group after selecting the group
+    also check error in production.log log file
+    Bugzilla:
+        1803952
+    Polarion:
+        assignee: dgaikwad
+        casecomponent: Configuration
+        caseimportance: critical
+        initialEstimate: 1/4h
+        testSteps:
+            1. Go to Configuration--> Acesss controls
+            2. Select existing custom group and Configure
+        expectedResults:
+            1.
+            2. Able to see Edit Group option
+    """
+    view = navigate_to(appliance.collections.groups, "All")
+    msg = ".*FATAL.*Error caught: [NoMethodError].*"
+    with LogValidator("/var/www/miq/vmdb/log/production.log",
+                      failure_patterns=[msg]).waiting(timeout=120):
+
+        # Selecting first row from all group
+        row = view.table[0]
+        row[0].check()
+        assert row[0].checked
```

Comment:
Is this bit redundant? I think it is browser's job to keep the element checked once it was checked. We don't check these things and take them as granted working.

If there was a problem that this got unchecked by some javascript, then I believe we need some kind of wait + check or periodic checking of the checked state during some time interval anyway.

Reply:
Changes are done